### DISCUSSION TITLE
Feature: Display Emoji as Icon if it's placed first in the collection name

### DISCUSSION THREAD

**Discussion by dom6770** [outline/outline#4162] [Ideas]
This is actually a feature from Microsoft's "To Do" app. Whenever you place an Emoji as a first character in the title, it replaces the symbol in front of the title. I think this would be a nice addition for more possible icons. 

Here's a video showing how it works in Microsoft To Do:

https://user-images.githubusercontent.com/11932291/191987867-a776e391-f734-4a94-b290-f3430744e2e2.mp4

**Comment by tommoor** [comment]
For documents this will already work if you do `emoji<space>title`, for collections we will at some point enable emojis as icons – amazingly not a request that has come up too often.

**Reply by Limezy** [reply]
I want it ! Funny enough I was about to post a similar request when that one came in :)

**Reply by almereyda** [reply]
Yes! Choosing an emoji icon is the first thing that I do in Nextcloud Collectives or Mattermost Boards. They are a universal visual language, that is abundant and widely recognised.

Please also see this previous discussion on "subcollections" a.k.a. documents with an emoji in their title that have subdocuments:

- https://github.com/outline/outline/discussions/3591#discussioncomment-2845420

The emoji will even be displayed apart in front of the document title:

![Bildschirmfoto vom 2022-09-26 01-27-20](https://user-images.githubusercontent.com/1645308/192170437-d5f94cb5-bcd6-4b54-8016-fb847a691ddd.png)

> With a little positioning issue, which leads to the word beginning slightly to the right from the block below it. Without icon, they are aligned.
>
> ![outline-headline-emoji-example](https://user-images.githubusercontent.com/1645308/192170991-893d3f1f-0d1d-473b-b1a4-9e5c822f557d.png)
>
> Maybe the titles beginning with an emoji could be parsed, in so text and emoji can be displayed independently from each other? It is already recognised as a special entity, as it is displayed more to the left. An additional tweak at this place will improve readability a lot.
>
> ![outline-headline-example](https://user-images.githubusercontent.com/1645308/192170985-ba96698d-e512-4ffa-90d8-17453f2ce70a.png)
>
> Do you see the difference?

There was also another discussion about collection icons here:

- https://github.com/outline/outline/discussions/1851
